src/elements/content-picker/ContentPicker.js

```diff
@@ -940,7 +938,7 @@ class ContentPicker extends Component<Props, State> {
         // if no shared link currently exists, create a shared link with enterprise default
         if (!item[FIELD_SHARED_LINK]) {
             // $FlowFixMe
-            this.changeShareAccess(null, item);
+            this.changeShareAccess(undefined, item);
```

Comment:
using `undefined` over `null` because of this comment: https://github.com/box/box-ui-elements/blob/5909f1abe2ec3701531fcbcb780c8a3cce9c7545/src/api/Item.js#L328

HOWEVER, I think both actually work so not sure if I should change everything to `null` so it's more intentional than `undefined`

Reply:
I think undefined actually makes sense here since we're making it fall back to the default. I can see `access = null` being ambiguous

Reply:
Should `access` in Item.js be of type `Access` as defined [here](https://github.com/box/box-ui-elements/blob/5909f1abe2ec3701531fcbcb780c8a3cce9c7545/src/common/types/core.js#L78)?